src/main/java/org/candlepin/insights/resource/InventoriesResource.java

```diff
@@ -43,20 +38,17 @@
 @Validated
 public class InventoriesResource implements InventoriesApi {
 
-    @Autowired
-    private InventoryController inventoryController;
+    private final InventoryController inventoryController;
+    private final TaskManager tasks;
 
-    @Autowired
-    private TaskManager tasks;
+    public InventoriesResource(InventoryController inventoryController, TaskManager tasks) {
```

Comment:
How are these getting injected without the `Autowired` annotation?

Reply:
Per [spring docs](https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-spring-beans-and-dependency-injection.html), if the bean has only one constructor, `@Autowired` can be omitted for constructor injection. I'm not opposed to adding `@Autowired` for clarity if you'd like though?

Reply:
Interesting!  No need to add it.